tools/rum/slicer.js

```diff
@@ -30,10 +31,29 @@ const herochart = window.slicer && window.slicer.Chart
   : new CWVTimeLineChart(dataChunks, elems);
 const sidebar = new FacetSidebar(dataChunks, elems);
 
+function extractedConversionParams() {
+  const { searchParams: params } = new URL(window.location);
+  const selectedCheckpoint = params.get('conversion.checkpoint');
+  const selectedSource = params.get('conversion.source');
+  const selectedTarget = params.get('conversion.target');
+  return [
+    {
+      checkpoint: selectedCheckpoint || 'click',
+      source: selectedSource,
+      target: selectedTarget,
+    },
+  ];
+}
+
```

Comment:
I'd make this a bit more flexible:
```
Array.from(params.entries).
  .filter( // starts with conversion.
  .map( // strip out conversion
``

then refactor the `updateFilter` function, so that you can re-use the main logic to extract a filter spec.

then re-use the same filter function in `cruncher.js` to apply the filter according to the spec (may need some refactoring, too)

Reply:
Thanks for spoting that filtering does not work at all with conversions :)